cpp/translator.cc

```diff
@@ -243,11 +243,40 @@ bool IsEmptyEntryWithoutTranslator(const std::string &entry) {
 }
 
 
+void SetupVuFindSortLimit(DbConnection &db_connection, std::string * const create_sort_limit, const std::string &offset,
+                          const bool use_untranslated_filter)
+{
+    if (use_untranslated_filter) {
+        const std::string translator=GetTranslatorOrEmptyString();
+        if (translator.empty()) {
+            ShowErrorPage("Error - No Valid User", "Not valid user selected");
+            std::exit(0);
```

Comment:
Why a non-error return after an error?  Also, we prefer to use EXIT_SUCCESS and EXIT_FAILURE for readability.